src/gluonts/evaluation/backtest.py

```diff
@@ -34,13 +34,16 @@
 
 
 def make_evaluation_predictions(
-    dataset: Dataset, predictor: Predictor, num_samples: int
+    dataset: Dataset,
+    predictor: Predictor,
+    num_samples: Optional[int] = 100,
```

Comment:
Is this really `Optional`?

Reply:
Some predictor may not need it. But maybe that's in the sense of "not have it passed at all", so this could just be an `int`.